src/components/LanguageProvider/LanguageProvider.container.js

```diff
@@ -64,11 +65,19 @@ export class LanguageProvider extends Component {
   }
 
   fetchMessages(lang) {
+    const { changeLang, showNotification } = this.props;
+
     this.setState({ messages: null });
 
-    importTranslation(lang).then(messages => {
-      this.setState({ messages });
-    });
+    importTranslation(lang)
+      .then(messages => {
+        this.setState({ messages });
+      })
+      .catch(() => {
+        changeLang('en-US');
```

Comment:
literal string, should use `DEFAULT_LANG` variable instead (equals to 'en-US')

Reply:
Oops, my bad, didn't see it. I changed it.